File: onebot-sdk/src/main/java/cn/evole/onebot/sdk/event/notice/friend/FriendAddNoticeEvent.java
package cn.evole.onebot.sdk.event.notice.friend;

import cn.evole.onebot.sdk.event.notice.NoticeEvent;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

/**
 * Created on 2022/7/8.
 *
 * @author cnlimiter
 */
@Data
@NoArgsConstructor
@SuperBuilder(toBuilder = true)
@EqualsAndHashCode(callSuper = true)
public class FriendAddNoticeEvent extends NoticeEvent {

}

File: onebot/src/main/kotlin/sdk/event/notice/friend/FriendAddNoticeEvent.kt
package cn.evolvefield.onebot.sdk.event.notice.friend

import cn.evolvefield.onebot.sdk.event.request.RequestEvent
import lombok.Data
import lombok.EqualsAndHashCode
import lombok.NoArgsConstructor
import lombok.experimental.SuperBuilder

@Data
@NoArgsConstructor
@SuperBuilder(toBuilder = true)
@EqualsAndHashCode(callSuper = true)
class FriendAddNoticeEvent : RequestEvent()
